packages/cli/src/commands/init/templates/github/workflows/novu.yml

```diff
@@ -12,7 +12,7 @@ jobs:
         uses: novuhq/actions-novu-sync@v2
         with:
           # The secret key used to authenticate with Novu Cloud
-          # To get the secret key, go to https://web.novu.co/api-keys.
+          # To get the secret key, go to https://dashboard.novu.co/api-keys.
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Nice catch!‏